source/playbooks/CIS/lib/cis-stack.ts

```diff
@@ -106,6 +106,29 @@ export class CisStack extends cdk.Stack {
     });
     cisPerms.addDependsOn(cis15111playbook.lambdaRole.node.findChild('Resource') as CfnRole)
 
+    //---------------------------------------------------------------------
+    // CIS116
+    //
+    let cis116findings = [
+      "1.16 Ensure IAM policies are attached only to groups or roles",
+    ];
+
+    const cis116playbook: PlaybookConstruct = new PlaybookConstruct(this, 'cis116playbook', {
+      name: 'CIS116',
+      description: 'Remediates CIS 1.16 by adding the users to a group with relevant polices, removing policies attached to them.',
```

Comment:
maybe:
`Remediates CIS 1.16 by adding users to a new group with polices obtained from the users.`

Reply:
Ok

Reply:
Fixed